site/public/js/ta-grading.js

```diff
@@ -406,7 +406,7 @@ function adjustGradingPanelHeader () {
   }
   // From the complete content remove the height occupied by navigation-bar and panel-header element
   // 6 is used for adding some space in the bottom
```

Comment:
This comment doesn't look relevant anymore and can be deleted

Reply:
Fixed.